src/test/java/com/teragrep/cfe_39/CombinedFullTest.java

```diff
@@ -0,0 +1,309 @@
+package com.teragrep.cfe_39;
+
+import com.teragrep.cfe_39.avro.SyslogRecord;
+import com.teragrep.cfe_39.consumers.kafka.KafkaController;
+import org.apache.avro.file.DataFileStream;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.*;
+import org.apache.hadoop.hdfs.DistributedFileSystem;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.hadoop.fs.Path;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.sql.Timestamp;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CombinedFullTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CombinedFullTest.class);
+    private static MiniDFSCluster hdfsCluster;
+    private static File baseDir;
+    private static Config config;
+
+
+    // Start minicluster and initialize config.
+    @BeforeAll
+    public static void startMiniCluster() throws IOException, InterruptedException {
```

Comment:
Tests should not throw nor try/catch anything, use `Assertions.assertDoesNotThrow`, `Assertions.assertThrows`, `Assertions.assertAll` and such instead

Reply:
Solved from commit 65342c2e865912f981dfe82f470cab6884f72cdb onward